sdk/apimanagement/Azure.ResourceManager.ApiManagement/tests/Azure.ResourceManager.ApiManagement.Tests.csproj

```diff
@@ -1,5 +1,35 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+  <ItemGroup>
+    <PackageReference Include="Azure.ResourceManager.EventHubs" OverrideVersion="1.0.0" />
+    <PackageReference Include="Azure.ResourceManager.Network" OverrideVersion="1.0.0" />
```

Comment:
Do we need the version here? These overridden versions are the same as in Packages.Data.Props

Reply:
Fxied